src/server_manager/electron_app/index.ts

```diff
@@ -193,6 +193,8 @@ function main() {
   // prevent window being garbage collected
   let mainWindow: Electron.BrowserWindow;
 
+  app.userAgentFallback = `OutlineManager/${electron.app.getVersion()} ${app.userAgentFallback}`;
```

Comment:
This is to allow the use of ngrok without the interstitial of the free version. But I also think it's helpful to tell we are using the Outline Manager.